de.dlr.sc.virsat.external.lib.jersey/build.properties

```diff
@@ -0,0 +1,37 @@
+source.. = src/
+output.. = target/classes/
+bin.includes = META-INF/,\
+               .,\
+               about.html,\
+               about_files/,\
+               externalLib/,\
+               externalLib/jaxrs-ri/api/jakarta.ws.rs-api-2.1.6.jar,\
+               externalLib/jaxrs-ri/ext/aopalliance-repackaged-2.6.1.jar,\
+               externalLib/jaxrs-ri/ext/hk2-api-2.6.1.jar,\
+               externalLib/jaxrs-ri/ext/hk2-locator-2.6.1.jar,\
+               externalLib/jaxrs-ri/ext/hk2-utils-2.6.1.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.activation-api-1.2.1.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.annotation-api-1.3.5.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.inject-2.6.1.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.json-1.1.5.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.json-api-1.1.5.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.json.bind-api-1.0.2.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.persistence-api-2.2.3.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.servlet-api-4.0.3.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.validation-api-2.0.2.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.ws.rs-api-2.1.6-sources.jar,\
+               externalLib/jaxrs-ri/ext/jakarta.xml.bind-api-2.3.2.jar,\
+               externalLib/jaxrs-ri/ext/javassist-3.25.0-GA.jar,\
+               externalLib/jaxrs-ri/ext/org.osgi.core-6.0.0.jar,\
+               externalLib/jaxrs-ri/ext/osgi-resource-locator-1.0.3.jar,\
+               externalLib/jaxrs-ri/ext/yasson-1.0.3.jar,\
+               externalLib/jaxrs-ri/lib/jersey-client.jar,\
+               externalLib/jaxrs-ri/lib/jersey-common.jar,\
+               externalLib/jaxrs-ri/lib/jersey-container-servlet-core.jar,\
+               externalLib/jaxrs-ri/lib/jersey-container-servlet.jar,\
+               externalLib/jaxrs-ri/lib/jersey-hk2.jar,\
+               externalLib/jaxrs-ri/lib/jersey-media-jaxb.jar,\
+               externalLib/jaxrs-ri/lib/jersey-media-json-binding.jar,\
+               externalLib/jaxrs-ri/lib/jersey-media-sse.jar,\
+               externalLib/jaxrs-ri/lib/jersey-server.jar
+	
```

Comment:
Could we add a new line at the end of the file?

Reply:
well, there is a new line already. What exactly do you mean?

Reply:
Looking to the other comments i guess its solved